infra/main.parameters.json

```diff
@@ -20,9 +20,18 @@
       "grafanaExists": {
         "value": "${SERVICE_GRAFANA_RESOURCE_EXISTS=false}"
       },
+      "elasticSearchImageName": {
+        "value": "${SERVICE_ELASTIC_SEARCH_IMAGE_NAME}"
+      },
+      "grafanaImageName": {
+        "value": "${SERVICE_GRAFANA_IMAGE_NAME}"
+      },
       "githubPat": {
         "value": "${GITHUB_PAT}"
       },
+      "githubOrganizationSlugs": {
```

Comment:
this is only used by the cpuadupdater env vars, if you want to pass it in as a parameter, that is fine, but you should use it and remove it from the env vars for the cpuadupdater in the main.parameters.json file

Reply:
this is to ensure AZD asks for it during deployment. 
with `azd up` is really easy to deploy the app without any organization slugs which leads to broken app